### ISSUE TITLE
KSType for inner types misses arguments from outer types

### ISSUE THREAD

**Issue by ting-yuan** [google/ksp#325]
Test case:
```
class Outer<T> {
    inner class Inner
}
val foo: Outer<String>.Inner
    get() = TODO()
```

**Comment by ting-yuan** [comment]
Fixed in #359

**Comment by yamasa** [comment]
@ting-yuan 
Thanks for addressing this. However, I find the behavior in `1.4.31-1.0.0-alpha06` very difficult to use.

I filed a new ticket to describe the problem.
https://github.com/google/ksp/issues/366
